README.md

```diff
@@ -27,12 +27,17 @@ It requires Node.js version 16 or higher.
 npm i && phet2zim
 ```
 
-The above will eventually output a ZIM file to ```dist/```
+The above will eventually output a ZIM file to ```output/```
 
 ## Command line arguments
 
 See `phet2zim --help` for details.
 
+`phet2zim --output` generates ZIM files in a specific folder.
+```bash
+phet2zim --output ZimFarm
```

Comment:
Please use another sample output name, `ZimFarm` is very confusing here for anyone not familiar with the tool, it looks like it might be something important while it's not